lib/modules/manager/ansible-galaxy/util.ts

```diff
@@ -5,7 +5,7 @@ export const blockLineRegEx = /^\s*((\w+):\s*(\S+))\s*$/;
 export const galaxyDepRegex = /[\w-]+\.[\w-]+/;
 export const dependencyRegex = /^dependencies:/;
 export const galaxyRegEx = regEx(
-  /^\s+(?<packageName>[\w.]+):\s*["'](?<version>.+)["']\s*/
+  /^\s+(?<packageName>[\w.]+):\s*["']?(?<version>.+?)["']?\s*(\s#.*)?$/
```

Comment:
```suggestion
  /^\s+(?<packageName>[\w.]+):\s*["']?(?<version>.+?)["']?\s*(#.*)?$/
```
Not necessary

Reply:
a valid yaml comment must have at least one whitespace in front of the '#' that's why i added the \s inside the regex group for the comment